### DISCUSSION TITLE
Send “favorite routes”.

### DISCUSSION THREAD

**Discussion by Leeb64** [devemux86/cruiser#460] [Q&A] [answered]
I would like to be able to send a route that is saved under: my places/routes via messenger (e.g. to one or more tour participants) without first saving it somewhere on my smartphone.
???

**Comment by devemux86** [comment, marked as answer]
@Leeb64 thanks for the idea.

You already suggested it in another topic, so let's continue the discussion there:
- https://github.com/devemux86/cruiser/discussions/330
